Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

```
@@ -48,7 +48,7 @@
         path: /Audio/Weapons/bladeslice.ogg
   - type: Item
     size: 15
-    sprite: Objects/Weapons/Melee/katana.rsi
+    sprite: DeltaV/Objects/Weapons/Melee/katana.rsi
```

```suggestion
    sprite: DeltaV/Objects/Weapons/Melee/katana.rsi #DeltaV
```

Missing a comment on an upstream file.